src/igvfd/schemas/technical_sample.json

```diff
@@ -8,7 +8,8 @@
         "award",
         "lab",
         "source",
-        "sample_material"
+        "sample_material",
+        "technical_sample_term"
```

Comment:
please add a test that tests that the term is required for posting new technical sample (test_schema...)